server/db_setup/create_schema.sql

```diff
@@ -17,4 +17,20 @@ CREATE TABLE FileTypeMap (
     file_type varchar(255),    
     upload_date DATETIME,
     is_uploaded boolean DEFAULT 0
+);
+
+DROP TABLE IF EXISTS User_File_SocketMap;
+
+CREATE TABLE User_File_SocketMap (    
+    user_name varchar(255),
+    file_unique_id varchar(36) NOT NULL,
+    current_socket_id varchar(255),
+    current_state varchar(255) 
+);
+
+DROP TABLE IF EXISTS User_id_Map;
+
+CREATE TABLE User_id_Map (
+    user varchar(255),
+    user_id varchar(255)
```

Comment:
The `User_id_Map` table should also have a primary key to uniquely identify each user. Consider adding a primary key constraint on `user_id` to ensure that user IDs are unique in the table.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*